webpack/scenes/Hosts/ChangeContentSource/index.js

```diff
@@ -86,45 +92,68 @@ const ChangeContentSourcePage = () => {
 
-  if (getHostIds(urlParams.host_id).length === 0 && urlParams.searchParam === '') {
-    return (
-      <Grid className="margin-40">
-        <GridItem span={7}>
-          <Alert
-            variant="danger"
-            title={__('No hosts with content source found!')}
-          />
-        </GridItem>
-      </Grid>);
-  }
-
   return (
-    <Grid className="margin-40">
-      <GridItem span={7}>
-        <h1>{__('Change host content source')}</h1>
-      </GridItem>
-      <Hosts
-        contentHosts={contentHosts}
-        hostsWithoutContent={hostsWithoutContent}
-      />
-
-      <ContentSourceForm
-        handleSubmit={handleSubmit}
-        environments={environments}
-        handleEnvironment={handleEnvironment}
-        environmentId={environmentId}
-        contentViews={contentViews}
-        handleContentView={setContentViewId}
-        contentViewId={contentViewId}
-        contentSources={contentSources}
-        contentSourceId={contentSourceId}
-        handleContentSource={handleContentSource}
-        contentHosts={contentHosts}
-        isLoading={isLoading}
-      />
-      { apiChangeStatus === STATUS.RESOLVED &&
-      <ContentSourceTemplate template={template} jobInvocationPath={jobInvocationPath} /> }
-    </Grid>
+    <>
+      <Head>
+        <title>{__('Change host content source')}</title>
+      </Head>
+      <PageSection
+        isFilled
+        variant="light"
+      >
+        <div className="margin-left-20">
+          <BreadcrumbBar
+            breadcrumbItems={[
+              { caption: __('Hosts'), url: foremanUrl('/hosts') },
+              { caption: 'Change host content source' },
+            ]}
+          />
+        </div>
+        <Grid className="margin-left-20">
+          <GridItem span={7}>
+            <Title
+              headingLevel="h5"
+              size="2xl"
+            >
+              {__('Change host content source')}
+            </Title>
+          </GridItem>
+          {noSearch &&
+            <GridItem span={7}>
+              <Alert
+                variant="danger"
+                className="margin-top-20"
+                title={__('No hosts with content source found!')}
+              />
+            </GridItem>
+        }
+          { !noSearch &&
+          <>
+            <Hosts
+              contentHosts={contentHosts}
+              hostsWithoutContent={hostsWithoutContent}
+            />
+
+            <ContentSourceForm
+              handleSubmit={handleSubmit}
+              environments={environments}
+              handleEnvironment={handleEnvironment}
+              environmentId={environmentId}
+              contentViews={contentViews}
+              handleContentView={setContentViewId}
+              contentViewId={contentViewId}
+              contentSources={contentSources}
+              contentSourceId={contentSourceId}
+              handleContentSource={handleContentSource}
+              contentHosts={contentHosts}
```

Comment:
When I change `host_id=` to a bogus ID, I get a `<ContentSourceForm>` but everything is disabled. That case should probably display the error alert instead.